ProcessLib/SmallDeformation/CreateLocalAssemblers.h

```diff
@@ -30,7 +30,6 @@ template <unsigned GlobalDim, int DisplacementDim,
 void createLocalAssemblers(
     NumLib::LocalToGlobalIndexMap const& dof_table,
     std::vector<MeshLib::Element*> const& mesh_elements,
-    unsigned const integration_order,
```

Comment:
The integration_order is removed because it's not used in createlocalassemblers, but it is still in the concrete local assemblers, isn't it?